handlers/group.ts

```diff
@@ -11,46 +12,49 @@ interface IGroup {
 
 class GroupHandle implements IGroup {
     readonly gun: Gun
-    constructor(private db: Gun) {
+    constructor(db: Gun) {
         this.gun = db
     }
 
     public create(groupName: string, callback : () => void): void {
         let user = this.gun.user(userPub)
-        let groupId : string = randomUUID();        
-        this.gun.get("groups").get("groupId").set({groupId: groupId})
+        const groupId : string = uuid();
+        this.gun.get("groups").get("groupId").set(groupId)
```

Comment:
this can also be written as `this.gun.get("groups").set({groupId: groupId})` I'm pretty sure, if we wanna keep the same style

Reply:
Så vidt jeg kan se bliver den syntaks ikke brugt nogle steder længere.
Tænker også det er at komplicere det uden nogen reel årsag.